sdks/python/apache_beam/examples/complete/estimate_pi_test.py

```diff
@@ -47,5 +47,4 @@ def test_basics(self):
 
 
 if __name__ == '__main__':
-  logging.getLogger().setLevel(logging.INFO)
```

Comment:
I prefer INFO to be the default here and elsewhere. Some tests might print useful information. For example I'm printing INFO logs in source test framework - https://github.com/apache/incubator-beam/pull/667/files

I think if we don't set it will default to WARN.